### PR TITLE
Add comments and headlines trials

### PR DESCRIPTION
**Pull request by Gwen-HH**

### CODE REVIEW

File: _posts/2019/2019-09-11-IbelieveIcanfly.md [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+header_position: center
+header_overlay: true
+category: on-the-job
+tags: ["company", "communication", "meeting"]
+authors: ["Gwen"]
+about_authors: ["gsahm"]
+---
+
+it's great to see all colleagues again and have some time together.
+#Trying the headlines!
```
Comment (shollerhh) [inline _posts/2019/2019-09-11-IbelieveIcanfly.md]: Why don't you write an actual headline?

**Comment by maletzke** [comment]
Nice!

File: _posts/2019/2019-09-11-IbelieveIcanfly.md [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+header_image: private/townhall.jpg
+header_position: center
+header_overlay: true
+category: on-the-job
+tags: ["company", "communication", "meeting"]
+authors: ["Gwen"]
+about_authors: ["gsahm"]
+---
+
+it's great to see all colleagues again and have some time together.
```
Comment (shollerhh) [inline _posts/2019/2019-09-11-IbelieveIcanfly.md]: I agree ;-)

File: _posts/2019/2019-09-11-IbelieveIcanfly.md [hunk 3]
```diff
@@ -0,0 +1,17 @@
...
+#Trying the headlines!
+##Trying the headlines again and again
```
Comment (somtoZikora) [inline _posts/2019/2019-09-11-IbelieveIcanfly.md]: You should leave a space between the hashtags and the title text so it's rendered properly in the blo post. 😉

**Pull request closed without merging**